src/components/dashboard/DoctorList.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+// import { IoLogoTwitter } from 'react-icons/io';
+// import { FaLinkedinIn, FaFacebookF } from 'react-icons/fa';
+import { useSelector, useDispatch } from 'react-redux';
+import { singleDoctor } from '../../redux/actions/DoctorActions';
+import styles from '../scss/Home.module.scss';
+// import { doctordata } from '../../mockdata';
+
+// const DoctorList = () => (
+//   <section className={styles['doctors-section']}>
+//      <ul className={styles['doctor-lists']}>
+//       {
+//         doctordata.map((item) => (
+//           <li className={styles.list} key={item.id}>
+//             <button type="button">
+//               Appointment
+//             </button>
+//             {/* <ul className={styles['icon-links']}>
+//               <li className={styles.link}><FaFacebookF /></li>
+//               <li className={styles.link}><IoLogoTwitter /></li>
+//               <li className={styles.link}><FaLinkedinIn /></li>
+//             </ul> */}
+//           </li>
```

Comment:
Like I said in backend component of this project, it's better to remove commented out code especially as you head into staging and production in real world.